data_updates/00009_20230116-145407_products.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : petr
+# Creation: 2023-01-16 14:54
+
+from superdesk.commands.data_updates import DataUpdate as _DataUpdate
+
+
+class DataUpdate(_DataUpdate):
+
+    resource = "products"
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        db = mongodb_database
+        products = list(db.products.find({}))
```

Comment:
Should we also remove `companies` from the `products` resource?

Reply:
I kept it there instead of doing a rollback